spec/post_spec.rb

```diff
@@ -0,0 +1,68 @@
+require_relative './rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) do
+    User.create(
+      name: 'Loren',
+      photo: 'https://this-person-does-not-exist.com/en/download-page?image=genef4e9868ae582ca3061881b69d8fbeb1',
+      posts_counter: 0,
+      bio: 'I am Loren, and I love sport.'
+    )
+  end
+
+  subject do
+    Post.new(
+      title: 'My first post',
+      author: user,
+      text: 'The motivation to become developer is very big.',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+  end
+
+  before { subject.save }
+
+  describe 'Validations' do
```

Comment:
- [ ] As required in the [previous](https://github.com/MarkoKermi/Blog_app/pull/3/files#r1186088995) review it could be better if you implement the test for the title exceeding 250 characters as required :wink:

Reply:
Well resolved ✔️